scripts/devcontainer_setup.sh

```diff
@@ -0,0 +1,21 @@
+# postCreateCommands for devcontainer.json
+
+# Ensure that the node modules directory is writable.
+sudo chown mesop-dev:mesop-dev node_modules
+
```

Comment:
fwiw, I think generally you don't need to run this, but it doesn't hurt either.

Reply:
The issue here is that Docker volume that gets created is owned by root. There does not appear to be a way to change this in the configuration since the node_modules directory gets created on the bind mount on the host with the source code.

If it wasn't on the bind mount, we could create the directory and change permissions ahead of time like we do with the vs code extensions  volume.